source/frontend/src/features/mapSideBar/shared/tabs/checklist/detail/ChecklistView.tsx

```diff
@@ -91,11 +91,11 @@ export const ChecklistView: React.FC<IChecklistViewProps> = ({
                 key={checklistItem.itemType?.code ?? `${prefix}-checklist-item-${j}`}
                 label={checklistItem.itemType?.description ?? ''}
                 tooltip={checklistItem.itemType?.hint}
-                labelWidth="6"
+                labelWidth={{ xl: 6, xs: 5 }}
```

Comment:
minor changes for smaller screens, this generally ensures that columns do not draw overtop of each other.